lib/ReactViews/Workbench/Controls/DimensionSelectorSection.tsx

```diff
@@ -62,7 +63,7 @@ class DimensionSelectorSection extends React.Component<PropsType> {
             <Select
               light
               name={dim.id}
-              id={dim.id}
+              id={`${this.props.item.uniqueId}-${dim.id}`}
```

Comment:
Duplicate ids across multiple catalog items were changing Select values - now they are not